client/my-sites/stats/stats-notices/commercial-site-upgrade-notice.tsx

```diff
@@ -20,22 +19,7 @@ const getStatsPurchaseURL = ( siteId: number | null, isOdysseyStats: boolean ) =
 const CommercialSiteUpgradeNotice = ( { siteId, isOdysseyStats }: StatsNoticeProps ) => {
 	const translate = useTranslate();
 	const isWPCOMSite = useSelector( ( state ) => siteId && getIsSiteWPCOM( state, siteId ) );
-	const [ noticeDismissed, setNoticeDismissed ] = useState( false );
-	const { mutateAsync: postponeNoticeAsync } = useNoticeVisibilityMutation(
-		siteId,
-		'commercial_site_upgrade',
-		'postponed',
-		30 * 24 * 3600
-	);
-
-	const dismissNotice = () => {
-		isOdysseyStats
-			? recordTracksEvent( 'jetpack_odyssey_stats_commercial_site_upgrade_notice_dismissed' )
-			: recordTracksEvent( 'calypso_stats_commercial_site_upgrade_notice_dismissed' );
-
-		setNoticeDismissed( true );
-		postponeNoticeAsync();
-	};
+	const [ noticeDismissed ] = useState( false );
```

Comment:
I think we could get rid of the state entirely here - we always will have it in the revisons imo.